platform/logging/LogLevel.ts

```diff
@@ -0,0 +1,55 @@
+import { assertNever } from '../lib/utils';
+
+export type LogLevelId =
+  | 'all'
```

Comment:
`all` and `off` logging levels are quite handy when you want to disable specific logger entirely (`off`) and allow everything to be logged (`all`).